app/config/services.yml

```diff
@@ -1,83 +1,46 @@
 services:
-    # First we define some basic services to make these utilities available in
-    # the entire application
-    slugger:
-        class: AppBundle\Utils\Slugger
-
-    markdown:
-        class: AppBundle\Utils\Markdown
-
-    # These are the Twig extensions that create new filters and functions for
-    # using them in the templates
-    app.twig.app_extension:
-        public:    false
-        class:     AppBundle\Twig\AppExtension
-        arguments: ['@markdown', '%app_locales%']
-        tags:
-            - { name: twig.extension }
+    # default configuration for services in *this* file
+    _defaults:
+        # automatically injects dependencies in your services
+        autowire: true
+        # automatically registers your services as commands, event subscribers, etc.
+        autoconfigure: true
+        # this means you cannot fetch services directly from the container via $container->get()
+        # if you need to do this, you can override this setting on individual services
+        public: false
 
-    app.twig.intl_extension:
+    # makes classes in src/AppBundle available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    AppBundle\:
+        resource: '../../src/AppBundle/*'
+        # you can exclude directories or files
+        # but if a service is unused, it's removed anyway
+        exclude: '../../src/AppBundle/{Entity,Repository}'
+
+    # controllers are imported separately to make sure they're public
+    # and have a tag that allows actions to type-hint services
+    AppBundle\Controller\:
+        resource: '../../src/AppBundle/Controller'
+        public: true
+        tags: ['controller.service_arguments']
```

Comment:
You're right... but it's nice to be consistent with the official project skeleton